docs/docs/guides/web3_migration_guide/index.md

```diff
@@ -32,6 +32,8 @@ const web3 = new Web3();
 
 Passing callbacks to functions is no longer supported, except for event listeners.
 
+For that, for example, the way to subscribe and do event listening on blockchain events had changed in 4.x as you can find inside [**`web3.eth.subscribe` Migration Guide**](/docs/guides/web3_migration_guide/subscribe_migration_guide#subscribing-to-events).
```

Comment:
```suggestion
For more information and examples on how to subscribe and listen to events emitted from smart contracts, and how doing so has changed in 4.x, please refer to the [**`web3.eth.subscribe` Migration Guide**](/docs/guides/web3_migration_guide/subscribe_migration_guide#subscribing-to-events).
```

Reply:
Thanks,
Actually, it is not just for ` events emitted from smart contracts` since we have the events for new blocks and others. Additionally, I did not want the reader to limit the functionality to only blockchain events. And to emphasize this, I now added a new paragraph.